drivers/watchdog/wdt_iwdg_stm32.c

```diff
@@ -19,7 +19,11 @@
 #include "wdt_iwdg_stm32.h"
 
 #define IWDG_PRESCALER_MIN	(4U)
+#if defined(CONFIG_SOC_SERIES_STM32U5X)
+#define IWDG_PRESCALER_MAX	(1024U)
```

Comment:
maybe with LL_IWDG_PRESCALER_256 and  LL_IWDG_PRESCALER_1024  definition from LL in a 
#ifdef LL_IWDG_PRESCALER_1024 
...